Vendor/tvOS/OHHTTPStubs/OHHTTPStubs.framework/Modules/module.modulemap

```diff
@@ -1,5 +1,5 @@
 framework module OHHTTPStubs {
-  umbrella header "OHHTTPStubsUmbrella.h"
+  umbrella header "OHHTTPStubs.h"
```

Comment:
Remove frameworks if you already have submodule (and do rebase to avoid keeping another manually built binary in thehistory)